src/components/ui/Checkboxes/Checkbox.jsx

```diff
@@ -19,6 +19,7 @@ function Checkbox({ checked, label, onChange }) {
       {label}
       <input
         onChange={handleChange}
+        defaultChecked={checked}
```

Comment:
why do you need this?

Reply:
that is right. I used value instead checked for checkbox :)